swagger.yaml

```diff
@@ -608,6 +608,68 @@ paths:
         "500":
           $ref: "#/components/responses/internal_server_error"
 
+  /blocks/{hash_or_number}/addresses:
+    get:
+      tags:
+        - Cardano » Blocks
+      summary: Addresses affected in specified block.
+      description: Return list of affected addresses in specified block with additional information.
+      parameters:
+        - in: path
+          name: hash_or_number
+          required: true
+          schema:
+            type: string
+            format: 64-character case-sensitive hexadecimal string or block number.
+          description: Hash of the requested block.
+          example: "4873401"
+        - in: query
+          name: count
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+            default: 100
+          description: The number of results displayed on one page.
+        - in: query
+          name: page
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 21474836
+            default: 1
+          description: The page number for listing the results.
+        - in: query
+          name: order
+          required: false
+          schema:
+            type: string
+            enum: [asc, desc]
+            default: asc
+          description: |
+            Ordered by hex value of an address.
```

Comment:
```suggestion
```
Ordering doesn't make sense here, it would be beneficial to note somewhere that it will be ordered by address first and then transactions are ordered by index.